codecov.yml

```diff
@@ -1,4 +1,3 @@
 ignore:
   - Pods
-  - Source/Alamofire
   - Source/PhoneNumberKit
```

Comment:
can we remove `Source/PhoneNumberKit` as well? I don't think this exists anymore either

Reply:
Due to the size of this work, I don't want to further increase the scope.

I think that changes to this integration should be undertaken separately, particularly since it appears that the integration itself is no longer functional.

Reply:
There is no scope to add! The folder, `Source/PhoneNumberKit` no longer exists. Delete line 3
```
  - Source/PhoneNumberKit
```

Reply:
Agree a new PR will to just remove this will help.

Reply:
I will raise a ticket to address this issue, including the occurrences in `.codebeatignore`, `.swiftlint.yml`, and `.jazzy.yml` as well 😄